code/test/e2e/loggedin/logger.spec.ts

```diff
@@ -7,10 +7,28 @@ test.use({
 
 test.beforeAll(({}, { config }) => {
   const { baseURL } = config.projects[0].use;
-  const mockedNetwork = fs.readFileSync('./test/e2e/loggedin/logger_base.har', { encoding: 'utf-8' });
+  let mockedNetwork = fs.readFileSync('./test/e2e/loggedin/logger_base.har', { encoding: 'utf-8' });
+  const expiryDate = new Date(new Date().getTime() + 1000 * 60 * 60 * 24 * 7).toISOString();
```

Comment:
Just a suggestion. It's simpler to fake the system time perhaps. We added a library for that and we used it in some tests.

Reply:
Good idea, but did we do that in e2e tests? I thought only for the date utils unit tests.

Going to check later